parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -614,6 +616,17 @@ should_skip_binary_cleanup() {
     echo "${should_skip,,}"
 }
 
+should_enforce_kube_pmc_install() {
+    set -x
+    body=$(curl -fsSL -H "Metadata: true" --noproxy "*" "http://169.254.169.254/metadata/instance?api-version=2021-02-01")
+    ret=$?
+    if [ "$ret" -ne 0 ]; then
+      return $ret
+    fi
+    should_enforce=$(echo "$body" | jq -r '.compute.tagsList[] | select(.name == "ShouldEnforceKubePMCInstall") | .value')
```

Comment:
How is ShouldEnforceKubePMCInstall assigned? and what's the purpose of using ShouldEnforceKubePMCInstall to determine the behavior of how to install kubelet?